src/Tests/TCPSmokeTests.cs

```diff
@@ -38,19 +38,20 @@ public void TearDownUdpListener()
 			// Stop listening for client requests.
 			tcpListener.Stop();
 		}
+        //this test requires a listener for a tcp socket...is it really a unit test?
```

Comment:
Nope - it's more integration. To be fair it never did say it was a unit test ;)  We just lumped them all together.